openfl/component/collaborator/collaborator.py

```diff
@@ -588,3 +608,41 @@ def named_tensor_to_nparray(self, named_tensor):
         self.tensor_db.cache_tensor({decompressed_tensor_key: decompressed_nparray})
 
         return decompressed_nparray
+
+    def get_memory_usage(self, round_number, metric_origin):
```

Comment:
It is good to have the overall memory usage, but if we can target the processes/containers specific to openfl, it will give more clear picture.

Reply:
This is the limitation currently. We will discuss this.